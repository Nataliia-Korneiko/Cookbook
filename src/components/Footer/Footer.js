import React from 'react';
import s from './Footer.module.css';

const Footer = () => {
  return (
    <>
      <footer className={s.footer}>
        <div className={s.container}>
          <p className={s.footer__title}>
            &#169; CookBook&#44; 2020 &#8226; All Rights Reserved &#8226; Design
            by Nataliia Korneiko
          </p>
        </div>
      </footer>
    </>
  );
};

export default Footer;
