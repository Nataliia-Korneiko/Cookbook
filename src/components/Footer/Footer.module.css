.container {
  margin: 0 auto;
  width: 1260px;
}

.footer {
  padding-top: 20px;
  padding-bottom: 20px;
  flex-shrink: 0;
}

.footer__title {
  margin: 0 auto;
  font-size: 14px;
  color: var(--color-golden);
  font-weight: 400;
  text-align: center;
  text-shadow: var(--text-shadow);
}

@media (max-width: 1300px) {
  .container {
    width: 940px;
  }
}

@media (max-width: 980px) {
  .container {
    width: 620px;
  }
}

@media (max-width: 660px) {
  .container {
    width: 300px;
  }
}
