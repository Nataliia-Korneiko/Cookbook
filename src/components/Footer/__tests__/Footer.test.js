import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

describe('Footer component', () => {
  const newFooter = shallow(<Footer />);

  it('renders a properly', () => {
    expect(newFooter).toMatchSnapshot();
  });

  it('renders one footer', () => {
    expect(newFooter.find('footer')).toHaveLength(1);
  });

  it('renders one div', () => {
    expect(newFooter.find('div')).toHaveLength(1);
  });

  it('renders one p', () => {
    expect(newFooter.find('p')).toHaveLength(1);
  });
});
