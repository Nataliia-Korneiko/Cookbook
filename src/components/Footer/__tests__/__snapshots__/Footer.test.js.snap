// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a properly 1`] = `ShallowWrapper {}`;
